website/tests/pages/user/group/index.spec.ts

```diff
@@ -25,4 +25,48 @@ test.describe('The group page', () => {
 
         await groupPage.verifyUserIsNotPresent(testUser);
     });
+
+    test('should edit group info and see changes afterwards', async ({ groupPage, loginAsTestUser }) => {
+        await loginAsTestUser();
+
+        await groupPage.goToUserPage();
+        await groupPage.goToGroupCreationPage();
+
+        const groupName = v4();
+        await groupPage.createGroup(groupName);
+        const newName = v4();
+        const newInstitution = v4();
+        const newEmail = `${v4()}@example.com`;
+        const newCountry = 2;
```

Comment:
There must be a better way to link list item 2 and Albania

Reply:
https://github.com/loculus-project/loculus/pull/2854/commits/e52ee6f1fede21ca4193024792176cbc8057f631